python/dolfin/io.py

```diff
@@ -206,6 +206,13 @@ def read_mesh(self, ghost_mode):
         mesh.geometry.coord_mapping = fem.create_coordinate_map(mesh)
         return mesh
 
+    def read_mesh_data(self, mpi_comm):
```

Comment:
Not clear from the doc string what the return type is.

Reply:
Changed the docstring.
Should type hints for return also be added? 
eg:
`->typing.Tuple[cpp.mesh.CellType, numpy.ndarray, numpy.ndarray,                                         typing.List[int]]:`

Reply:
Yes, that would be good.

If we use type hints, we should remove the types from the docstring to avoid duplication.